src/odb/include/odb/db.h

```diff
@@ -1368,6 +1369,11 @@ class dbBlock : public dbObject
 
  private:
   void ComputeBBox();
+  std::string makeNewName(dbModInst* parent,
```

Comment:
It is better to put private methods on the _dbBlock class (ComputeBBox should be the same).